tsdb/engine/wal/wal.go

```diff
@@ -455,7 +458,7 @@ func (l *Log) nextMetaFile() error {
 			return err
 		}
 
-		id = int(n)
+		id = int(n) + 1
```

Comment:
we roll over to a new meta file that can be written to while the old ones get emptied out and flushed to the index